scripts/azure-pipelines.yml

```diff
@@ -311,5 +311,26 @@ jobs:
       ./scripts/install_system_deps.sh  # no intel python
       ./scripts/install_python_deps.sh  # numpy, conda-build and etc.
       echo ========================= build DPNP package ===============================
-      export DPNP_BACKEND_TESTS_ENABLE=1
       ./0.build.sh
+
+# DPNP backend clib tests building and running
+- job: test_dpnp_backend_clib_lin
+  displayName: test_dpnp_backend_clib_lin
+  pool:
+    vmImage: 'ubuntu-20.04'
+  steps:
+  - bash: |
+      . ./scripts/install_cmake_lin.sh
+      ./scripts/install_system_deps.sh  # no intel python
+      echo ========================= build DPNP package ===============================
+      . /opt/intel/oneapi/setvars.sh
+      THEDIR=$(pwd)
+      cmake  -S$THEDIR/dpnp/backend -B$THEDIR/build/ -DCMAKE_BUILD_TYPE=Release \
```

Comment:
make it simpler. Do not use unnecessary variables like `CMAKE_BUILD_TYPE` or `CMAKE_VERBOSE_MAKEFILE` and etc

Reply:
This is not so critical. But I can remove it if you insist.

Reply:
This is not so critical. But I can remove it if you insist.